clusters/build-clusters/01_cluster/openshift-monitoring/main_alertmanager.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Can't we just use the operator for this stuff in 4.x?

Reply:
This is the operator stuff ... but the default CR does not come with the PVC as storage.

```
$ oc get alertmanager -n openshift-monitoring
NAME   AGE
main   20d
```
What I did is ... get the default CR and add the storage part.
Hoping there is a better way.

We will see how upgrading of cluster/monitoring-stack will treat my customization.
I guess after the upgrading, I need to pick the change and merge my customization again (if we want to keep a copy of current CR).

Reply:
Looks like some of this is documented here? https://docs.openshift.com/container-platform/4.2/monitoring/cluster-monitoring/configuring-the-monitoring-stack.html#configuring-a-local-persistent-volume-claim_configuring-monitoring

Reply:
Done.
OCP 4 is great~~! No hacking ... at least not as much as 3.